packages/node_modules/express-pouchdb/lib/index.js

```diff
@@ -185,6 +185,38 @@ module.exports = function (startPouchDB, opts) {
     next();
   });
 
+  app.use(function (req, res, next) {
+    var regexFindDbName = /\/([^\/]*)/
+    var dbName = decodeURIComponent(req.url.match(regexFindDbName)[1])
+
+    if (!dbName) {
+      return next();
+    }
+
+    // check for internal routes GET /_all_dbs
+    if (dbName[0] === '_') {
+      for (var i = 0, route; i < app._router.stack.length; i++) {
+        route = app._router.stack[i].route;
+        if (route) for (var j = 0, method; j < route.stack.length; j++) {
+          method = route.stack[j].method.toUpperCase()
+          if (method === req.method && route.path.substr(1, dbName.length) === dbName) {
+            console.log('ok?')
+            return next();
+          }
+        };
+      }
+    }
```

Comment:
I would personally go with the way @nolanlawson suggested here: patching the PouchDB constructor. There is actually logic for that in place already: express-pouchdb already uses PouchDB.new() instead of new PouchDB() everywhere (or it should), which can be wrapped using pouchdb-wrappers.

Now on where to do that and how to set it up... Probably somewhere here: https://github.com/pouchdb/pouchdb-server/blob/master/packages/node_modules/express-pouchdb/lib/routes/db.js#L130 , where allDbs() is already being patched as well. Just also add an implementation for new() in that same object that's passed to wrappers.installStaticWrapperMethods.

I hope that works?